tests/phpunit/tests/Story_Renderer.php

```diff
@@ -17,18 +17,16 @@
 
 namespace Google\Web_Stories\Tests;
 
-use Google\Web_Stories\Discovery;
-
 class Story_Renderer extends \WP_UnitTestCase {
-	protected static $user;
-
-	public static function wpSetUpBeforeClass( $factory ) {
-		self::$user = $factory->user->create( [ 'role' => 'administrator' ] );
-	}
-
 	public function setUp() {
 		// Avoids HTML in post_content being stripped because of lacking capabilities.
-		wp_set_current_user( self::$user );
+		remove_filter( 'content_save_pre', 'wp_filter_post_kses' );
```

Comment:
Much more robust IMO

Reply:
I don't understand what you are doing here? Can you provide more context and add a comment please?

Reply:
There‘s already a comment on the line above.

KSES in WordPress is responsible for stripping HTML if the user doesn‘t have unfiltered_html caps.

In these tests here, we don‘t need KSES.

Reply:
Clarified the comment a bit more.

```php
// When running the tests, we don't have unfiltered_html capabilities.
// This change avoids HTML in post_content being stripped in our test posts because of KSES.
```